public/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Chat</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+    <div id="messages"></div>
+    <form action="">
+        <input id="username" type="text" name="firstname" placeholder="username">
+        <input id="m" autocomplete="off" placeholder="text" />
+        <button>Send</button>
+    </form>
+
+
+    <script src="/socket.io/socket.io.js"></script>
+    <script src="https://code.jquery.com/jquery-1.11.1.js"></script>
+    <script>
+        $(() => {
+            var socket = io();
+            $('form').submit(() => {
+                socket.emit('chat message', { username: $('#username').val(), text: $('#m').val() });
+                $('#m').val('');
+                $('#username').val('');
+                return false;
+            });
+            socket.on('chat message', msg => {
```

Comment:
Something to think about:
Instead of using the ugle else if,
what if, you used an OBJECT!
var actions = {
'/setColor': doSomthing,
'/setBold': doSomethingElse
}

and then you just do something like: actions[msg.split(' ')[0]]

Reply:
Yeah .. mgs.split('')[0] is also ugly